server/db/models/user.js

```diff
@@ -26,27 +29,35 @@ const user = (sequelize, DataTypes) => {
     },
     password: {
       type: DataTypes.STRING,
+      allowNull: true
     },
     bio: {
-      type: DataTypes.TEXT
+      type: DataTypes.TEXT,
+      allowNull: true
     },
     favouriteTags: {
-      type: DataTypes.ARRAY(DataTypes.STRING)
-    },
-    roleId: {
-      type: DataTypes.INTEGER
+      type: DataTypes.ARRAY(DataTypes.STRING),
+      allowNull: true
     },
+    // roleId: {
```

Comment:
#### Nitpick
if these lines of code are not needed please remove them

Reply:
Okay. okay. on it